practice_app/pages/index.js

```diff
@@ -198,14 +199,13 @@ export default function Home({ context }) {
     // current functionality : source_lang=english, target_lang=ip_dependent
     const getTranslation = async () => {
         try {
-            // Learn client's ip adress using a third party service
-            const ipInfo = await axios.get('https://json.geoiplookup.io/')
-            // Provide client's ip & original text to our api
+            const isoConv = require('iso-language-converter')
```

Comment:
require(..) should not be used in client side stuff. Import it at the top of the file like this
```js
import isoConv from 'iso-language-convert' // OR
import * as isoConv from 'iso-language-convert'
```
whichever works.

Reply:
they did not work so i moved the conversion to the server side